README.md

```diff
@@ -2633,7 +2649,7 @@ Other Style Guides
       inventorOf,
     );
 
-    // good (note that a comma must not appear after a "rest" element)
+    // good (note that a comma must not appear after a "spread" element)
```

Comment:
this is incorrect; "rest" and "spread" are two different contextual meanings for the operator composed of three dots - and the comma is forbidding after *rest*, not after spread.